### PR TITLE
Add docker build and push to travis

### PR DESCRIPTION
**Pull request by mortenlj**

### CODE REVIEW

**Comment by mortenlj** [comment]
Now that I have seen it build on Travis, I'm going to change it a bit. The matrix part really isn't that interesting, and I'd want to build the docker image on all builds, not just when deploying.

Expect some changes.

**Comment by birgirst** [comment]
> I'd want to build the docker image on all builds, not just when deploying.
but only push on deploy right?

**Comment by mortenlj** [comment]
> but only push on deploy right?

push/deploy only on master, build always. 

I've made the changes I wanted to do now, decided that when matrix was removed, we didn't need the stages thing (which is a beta feature of travis).

I also made it so that coverage is uploaded if the build succeeds, but we don't fail the build if uploading coverage reports fails. Also, we only run tests once.

File: bin/docker_push [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+set -evuo pipefail
+
+VERSION=$(python setup.py --version)
+DOCKER_IMAGE="fiaas/skipper:${VERSION/+/-}"
+
+echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
+
+docker tag "fiaas/skipper:latest" "${DOCKER_IMAGE}"
+
+if [[ "${CI}" == "true" ]]; then
```
Comment (fiunchinho) [inline bin/docker_push]: This variable is always true. You are checking the value for cases when you run it locally?

Comment (mortenlj) [reply bin/docker_push]: Yes. It made more sense to test the scripts this way.

**Comment by mortenlj** [comment]
> if you use TRAVIS_REPO_SLUG instead of repeating fiaas/skipper, you could reuse these files on other projects.

I was considering that, and discarded it because of some reason I no longer remember. Can't have been a good reason, so I'll change it.

**Pull request merged**